HelpSource/Classes/Image.schelp

```diff
@@ -182,8 +199,18 @@ code::
 Image.interpolations;
 ::
 
-METHOD::closeAllPlotWindows
-close all the Image plot windows currently opened.
+METHOD::resizeModes
+returns an link::Classes/Array:: of the different resize modes you can specify when changing the size of an Image.
+code::
+Image.resizeModes;
+::
+
+METHOD::allPlotWindows
+Returns an array of all the Image plot windows currently opened.
+code::
+Image.allPlotWindows
+::
```

Comment:
probably would make the most sense to place this above `closeAllPlotWindows`?

Reply:
I was thinking about that, but now this is in a subsection `Class variables and attributes`, while I put `closeAllPlotWindows` is in the "main" class methods section, as it performs an action. Do you think they should be together in the main section or in the "class variables and attributes" section?

Reply:
ah okay, it's fine i guess, thanks for the explanation!